ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -198,13 +189,13 @@ let on_dom0_networking_change ~__context =
         debug "Changing Host.address in database to: %s" ip;
         Db.Host.set_address ~__context ~self:localhost ~value:ip;
         debug "Refreshing console URIs";
-        update_getty ();
+        Helpers.update_getty ();
         Dbsync_master.refresh_console_urls ~__context
       end
     | None ->
       if Db.Host.get_address ~__context ~self:localhost <> "" then begin
         debug "Changing Host.address in database to: '' (host has no management IP address)";
-        update_getty ();
+        Helpers.update_getty ();
         Db.Host.set_address ~__context ~self:localhost ~value:""
```

Comment:
I think it's fine, as we call the `update-issue` script, which gets the management interface from the inventory file ` /etc/xensource-inventory`, and then gets the IP using `ip addr show dev $MANAGEMENT_INTERFACE`. So as long as we update the inventory before this call, it's ok.